### DISCUSSION TITLE
New Update - no longer showing our meetings - Time Zone

### DISCUSSION THREAD

**Discussion by aeb-denver-CO** [code4recovery/12-step-meeting-list#897] [Q&A] [answered]
Thank you for your service & creating this magnificent plug in that has been helpful to so many.
our website:  https://daccaa.org/meetings/

Since the last update our website is no longer showing all of the meetings for that day as the default. It is adjusting to what it thinks is the next meeting for the day, only we may be in a different time zone that what it is posting.  

From what I can tell the updated version of the plugin, it defaults to anyday, anytime, any type, anywhere but only posts what it believes to be the next meeting of the day:

![image](https://user-images.githubusercontent.com/90991939/184016614-40a3ae36-cbae-46c5-8afb-ddb3be999a06.png)

It is currently 2:45pm Mountain time. There is a 2pm meeting going on & several 3pm meeting - none are showing. 
It is saying our next meeting is at 5pm Mountain time. Also, has the search by district feature been completely removed? Our fellowship uses this often.  Please assist. 

I pasted a screen shot.

Thank you for your time.

Abby Hoke, Office Manager, DACCAA

**Comment by joshreisner** [comment, marked as answer]
hi @aeb-denver-CO ! sounds like it thinks it's on new york time. is the timezone on your site set correctly? check in Wordpress Dashboard > Settings

**Reply by joshreisner** [reply]
<img width="1114" alt="Screen Shot 2022-08-10 at 6 39 48 PM" src="https://user-images.githubusercontent.com/1551689/184050711-82dd806c-f814-451b-bae0-868b21be7b05.png">

**Reply by joshreisner** [reply]
glad to see it's fixed now

**Reply by aeb-denver-CO** [reply]
Excellent! Yes that worked, thank you for your expedient response & solution!
